src/frontend/js/media-management/component/edit-sidebar.tsx

```diff
@@ -305,6 +322,31 @@ export default function EditSidebar({
           )}
         </div>
       </form>
+
+      {/* Hidden form for file replacement */}
+      <form
+        onSubmit={submitForm}
+        action={apiEndpoints.replaceFile}
+        class="hidden"
+      >
+        <input name="id" type="hidden" value={file.id} />
+        <input
+          id="replace-file-input"
+          name="file"
+          type="file"
+          maxLength={255}
+          accept={changedFile.type}
+          style={{display: "none"}}
```

Comment:
```suggestion
```
In general: Please use `class="hidden"` instead of `style={{display: "none"}}`. In this case however, since the form isn't shown anyway, it's not required at all.